tests/bdd/volgroup_vgpattern_bdd.feature

```diff
@@ -0,0 +1,156 @@
+
+Feature: Validate vgpattern and volgroup based scheduling.
+  Background:
+      Given Single lvmnode cluster is configured
+      And lvm vg "lvmvg" is created on a node
+
+      Scenario: test sc with volgroup present on the node
+          Given a sc is created with volgroup as "lvmvg"
+          And pvc is created referencing the same sc
```

Comment:
let's use full forms of these abbreviations throughout `sc -> storage class` , `cr -> custom resource`, `pvc -> volume`

Reply:
Okay....will  keep pvc as is, making it volume might not be clearly referencing it to pvc.

Reply:
On the second thought, These all are k8s recognised shortforms. Could be okay to keep all as is??

Reply:
yeah, but bdds are kind of generic and not k8s specific. You can refer maybe Mayastor bdds, if they use similar terminology, let's keep that.